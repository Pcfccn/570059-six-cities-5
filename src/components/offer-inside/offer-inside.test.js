import React from "react";
import renderer from "react-test-renderer";
import Inside from "./offer-inside";

it(`Should Inside render correctly`, () => {
  const tree = renderer
    .create(<Inside
      inside={[`a`, `b`]}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
