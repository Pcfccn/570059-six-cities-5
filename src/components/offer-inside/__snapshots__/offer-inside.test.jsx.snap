// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Inside render correctly 1`] = `
Array [
  <li
    className="property__inside-item"
  >
    a
  </li>,
  <li
    className="property__inside-item"
  >
    b
  </li>,
]
`;
